### PR TITLE
chore: bump thrift to 0.17.0

### PR DESCRIPTION
**Pull request by fMeow**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,21 +6,21 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/midnightexigent/thrift-pool-rs"
 description = "Easily make Connection Pools compatible with r2d2 and bb8 from any Thrift Client"
 
-[package.metadata.docs.rs]
-all-features = true
+  [package.metadata.docs.rs]
+  all-features = true
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+  # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```
Comment (midnightexigent) [inline Cargo.toml]: indentation doesn't look right

File: Cargo.toml [hunk 2]
```diff
@@ -6,21 +6,21 @@ license = "MIT OR Apache-2.0"
 repository = "https://github.com/midnightexigent/thrift-pool-rs"
 description = "Easily make Connection Pools compatible with r2d2 and bb8 from any Thrift Client"
 
-[package.metadata.docs.rs]
-all-features = true
+  [package.metadata.docs.rs]
+  all-features = true
```
Comment (midnightexigent) [inline Cargo.toml]: indentation doesn't look right

**Review by midnightexigent — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 2 above)
Comment (fMeow) [reply Cargo.toml]: Sorry, the indentation is introduced by auto formatting with [taplo](https://github.com/tamasfe/taplo).

If this doesn't look right to you, I will revert it to only bump version for thrift.

Thanks for your fast response! :)

Comment (midnightexigent) [reply Cargo.toml]: formatting for this repo is already handled by [dprint](https://dprint.dev)

you can install it locally and run `dprint fmt`

**Comment by fMeow** [comment]
I will make a PR in the thrift-hbase repo after this PR is accepted and published to crates.io with a version bump.

**Review by midnightexigent — APPROVED** [review]

**Pull request merged**